### DISCUSSION TITLE
Help understanding scv.tl.rank_velocity_genes results, what do the scores mean?

### DISCUSSION THREAD

**Discussion by EvanLemire** [theislab/scvelo#477] [Q&A]
I have an analysis where i am trying to determine the difference between two sets of cells whose velocities are pointing them in different directions. I used rank_velocity_genes on the groups, my understanding was that this would determine which genes in each cluster were driving the velocities. It returned a result that made sense, but when I look at the scatters of spliced vs unspliced, I actually don't see much difference between genes at the top and bottom of the list fora given cluster. Many of the genes at the bottom of the list with negative scores appear to be induced based on these plots, as do some genes with scores around 0. I had thought that a positive score for gene ranking meant induction and negative meant repression but this seems to not be the case, how can I interpret the output of the gene ranking and what does the score correspond to?

**Comment by WeilerP** [comment]
@EvanLemire, from the docs for `scv.tl.rank_velocity_genes` [here](https://scvelo.readthedocs.io/scvelo.tl.rank_velocity_genes/#scvelo.tl.rank_velocity_genes):

> This applies a differential expression test (Welch t-test with overestimated variance to be conservative) on velocity expression, to find genes in a cluster that show dynamics that is transcriptionally regulated differently compared to all other clusters (e.g. induction in that cluster and homeostasis in remaining population).

As such, the returned values are the scores of the t-test.
